src/index.js

```diff
@@ -20,6 +20,7 @@ type Props = {|
   fadeOut?: boolean,
   autoStart?: boolean,
   autoStartDelay?: number,
+  renderToHardwareTextureAndroid?: boolean,
```

Comment:
same as above

> i'm not sure we have to make it customizable if it creates UI glitches ?
What do you think about setting it to true everytime and making it transparent for developers ?

Reply:
Updated!